modules/hp_video_slideshow/hp_video_slideshow.libraries.yml

```diff
@@ -0,0 +1,12 @@
+styling:
+  version: VERSION
+  css:
+    theme:
+      'build/css/styles.css': {}
+  js:
+    '//code.jquery.com/jquery-3.3.1.min.js': { type: external, minified: true }
+    '//cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js': { type: external, minified: true }
+    '//stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js': { type: external, minified: true }
+    '//stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js': { type: external, minified: true }
+    '//malsup.github.com/jquery.cycle.all.js': { type: external }
```

Comment:
Add to HU Gen theme.

Reply:
I suggest using  a preprocessor function in the module's .theme to call on these external files rather than adding them to the HU General Folder.

Reply:
1. This seems like a whole lot of new external libraries to load. Is there a reason we are adding the entire bootsrap library for this feature?
2. Bootsrap is added twice.
3. Is there a reason we don't want to use the carousel functionality already built into the theme? See static example hu_general/idfive-component-library/src/components/ks/carousel

Reply:
Meant to remove bootstrap, that was a copy paste from a theme I was working on. Only jquery, jquery Cycle, and scripts.js are needed.

@drogers98 are you suggesting using the carousel library for transitions? Does that library support using event triggers to transition to the next slide?